src/gpgmm/utils/ContiguousLinkNodeAllocator.h

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 The GPGMM Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GPGMM_COMMON_CONTIGUOUSLINKNODEALLOCATOR_H_
+#define GPGMM_COMMON_CONTIGUOUSLINKNODEALLOCATOR_H_
+
+#include "gpgmm/utils/Assert.h"
+#include "gpgmm/utils/LinkedList.h"
+
+#include <queue>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+namespace gpgmm {
+
+    // ContiguousLinkNodeAllocator is an allocator designed to store objects contiguously by storing
```

Comment:
Using multiple std::vector doesn't make it an allocator, it's just a container with stability. Please update the comment. If you decide to change this to follow std::allocator (ex. no initialization upon Allocate) then we can revisit.

Reply:
Please disregard per your earlier comment.